_includes/host-pane.njk

```diff
@@ -1,4 +1,5 @@
   <li class="pane">
+    <div class="episode">
```

Comment:
Is this necessary? If so, you need to update the `.pane > dl > dt` rule so that it matches again. Just drop the first `>`.

Reply:
i wrapped it in the div so i could make the 'extras section' have a different background

Reply:
removed